### DISCUSSION TITLE
Group attributes

### DISCUSSION THREAD

**Discussion by christianlemmers** [swellstores/community#105] [Help] [answered]
Hi y'all!

I'm currently setting up attributes in Swell and I'd like to group several attributes together. For instance; I'd like to bundle length, width, height and depth into an attribute group called "sizes". What would be the best way to achieve this?

I'm currently working with a custom text field added to the Attributes model but content editors have to manually fill out 'Sizes' for each attribute they want to add to this group.

Thanks!

**Comment by ericingram** [comment, marked as answer]
Swell doesn't have a native "attribute group" concept yet. Are you building a custom dashboard?

**Reply by christianlemmers** [reply]
Nope, we're trying to use the Swell UI as much as possible so a feature like this would be really helpful. Any ideas how to do this with custom fields?

**Reply by ericingram** [reply]
I think you can just add a custom field called `group`, for example:

<img width="1233" alt="image" src="https://user-images.githubusercontent.com/952134/214324034-f7e5c34e-4519-48b7-a7af-c523c00dd20a.png">

**Reply by christianlemmers** [reply]
Yeah that could probably help. Not the most flexible solution, but maybe flexible enough for this particular use case.
